common/common.cpp

```diff
@@ -804,10 +804,10 @@ std::vector<llama_token> llama_tokenize(
     // upper limit for the number of tokens
     int n_tokens = text.length() + add_bos;
     std::vector<llama_token> result(n_tokens);
-    n_tokens = llama_tokenize(ctx, text.c_str(), result.data(), result.size(), add_bos);
+    n_tokens = llama_tokenize(ctx, text.c_str(), text.length(), result.data(), result.size(), add_bos);
```

Comment:
```suggestion
    n_tokens = llama_tokenize(ctx, text.data(), text.length(), result.data(), result.size(), add_bos);
```

Reply:
Thanks! Will change.

Reply:
Stupid me didn't simply accept your changes, sorry!

Reply:
It's fine, it would have only applied to the first case anyway.